client/tests/AtVersions.test.js

```diff
@@ -0,0 +1,48 @@
+import getPage from './util/getPage';
+
+describe('AT Version UI', () => {
+    test('should add, edit, then remove an AtVersion', async () => {
+        /* prettier-ignore */
+        await getPage({ role: 'admin', url: '/test-queue' }, async page => {
+            const openTrayIfClosed = async () => {
+                await page.waitForSelector('button ::-p-text(Manage Assistive Technology Versions)');
+                const isTrayClosed = !!(await page.$('::-p-text(Select an Assistive Technology and manage its versions)'));
+                if (isTrayClosed) {
+                    await page.click('button ::-p-text(Manage Assistive Technology Versions)');
+                    await page.waitForSelector('::-p-text(Select an Assistive Technology and manage its versions)');
+                }
+            };
+            await openTrayIfClosed();
+            await page.click('button ::-p-text(Add a New Version)');
+            await page.waitForSelector('.modal-title ::-p-text(Add a New Version for JAWS)');
+            await page.waitForSelector('.modal-body .form-group:nth-child(1) ::-p-text(Version Number)');
+            await page.waitForSelector('.modal-body .form-group:nth-child(2) ::-p-text(Approximate date of availability)');
+            await page.type('.modal-body .form-group:nth-child(1) input', '99.0.1');
+            await page.type('.modal-body .form-group:nth-child(2) input', '01-01-2000');
+            await page.click('.modal-footer button ::-p-text(Add Version)');
+            await page.waitForNetworkIdle();
+            await openTrayIfClosed();
```

Comment:
Yes! I think it's caused by the page either refreshing the component or possibly refreshing the entire page. It would be great to follow up and address that.

Reply:
Certainly something to track!